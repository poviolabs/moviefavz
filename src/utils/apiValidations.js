const validResponse = (status) => status && status >= 200 && status < 300;

export default { validResponse };
