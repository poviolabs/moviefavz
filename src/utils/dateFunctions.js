import dayjs from 'dayjs';

const getUnixDay = (timestamp = Date.now()) => {
  return dayjs(timestamp).startOf('date').valueOf();
};

const fillDaysToMatch = (num, data) => {
  if (!data.length) {
    return [];
  }
  const [lastDate] = data.reverse();

  return Array.from('x'.repeat(num - data.length))
    .map((_, i) => i + 1)
    .reverse()
    .map((i) => {
      return {
        value: 0,
        timestamp: dayjs(lastDate.timestamp).subtract(i, 'day').valueOf(),
      };
    });
};

export default { getUnixDay, fillDaysToMatch };
