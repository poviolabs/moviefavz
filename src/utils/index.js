export { default as arrayFunction } from './arrayFunctions';
export { default as apiValidations } from './apiValidations';
