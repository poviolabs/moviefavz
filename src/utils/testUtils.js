import React from 'react';
import { render } from '@testing-library/react';

import { BrowserRouter } from 'react-router-dom';

import { ThemeProvider } from 'styled-components';
import { theme } from '../styles';

// eslint-disable-next-line react/prop-types
const AllTheProviders = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      <BrowserRouter>{children}</BrowserRouter>
    </ThemeProvider>
  );
};

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options });

// re-export everything
export * from '@testing-library/react';

// override render method
export { customRender as render };
