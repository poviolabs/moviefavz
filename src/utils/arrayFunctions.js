import dateFunctions from './dateFunctions';

const groupArr = (data, size) => {
  const group = [];
  for (let i = 0, j = 0; i < data.length; i++) {
    if (i >= size && i % size === 0) {
      j++;
    }
    group[j] = group[j] || [];
    group[j].push(data[i]);
  }
  return group;
};

const groupByDay = (data, key = 'timestamp') => {
  const groupedByKey = data.reduce((obj, item) => {
    const timestamp = item[key];
    const day = dateFunctions.getUnixDay(timestamp);

    if (Object.prototype.hasOwnProperty.call(obj, day)) {
      obj[day].value += 1;
    } else {
      obj[day] = {
        value: 1,
        timestamp,
      };
    }
    return obj;
  }, {});
  return Object.values(groupedByKey);
};

export default { groupArr, groupByDay };
