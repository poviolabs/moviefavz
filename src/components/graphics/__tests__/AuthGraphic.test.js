import React from 'react';
import { render } from '../../../utils/testUtils';

import AuthGraphic from '../AuthGraphic';

describe('<AuthGraphic />', () => {
  it('Renders the Component', () => {
    const props = {};
    const container = render(<AuthGraphic {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
