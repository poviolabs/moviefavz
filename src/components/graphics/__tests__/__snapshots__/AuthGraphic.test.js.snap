// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthGraphic /> Renders the Component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        data-name="Layer 1"
        height="265"
        viewBox="0 0 924.256 530.591"
        width="462"
      >
        <path
          d="M16.126 159.262a12.416 12.416 0 01-15.82-14.588C7.53 114.622 34.998 41.128 135.064 10.217c43.345-13.39 84.914-13.619 123.5-.685 31.744 10.596 52.817 27.178 62.897 35.193a12.313 12.313 0 012.017 17.296l-.076.095a12.603 12.603 0 01-17.56 1.994c-18.264-14.39-73.685-58.105-163.508-30.358-87.706 27.093-111.53 90.466-117.71 116.393a12.231 12.231 0 01-8.498 9.117z"
          fill="#db0000"
        />
        <path
          d="M288.866 329.481a12.346 12.346 0 01-3.067.577c-78.175 4.21-127.914-60.736-129.956-63.443l-.378-.625c-1.247-1.934-31.342-47.723-18.093-84.088 6.086-16.625 19.544-28.294 40.116-34.65 19.133-5.91 34.79-4.162 48.116 5.354 10.926 7.753 17.93 19.315 24.717 30.48 14.116 23.182 22.767 34.977 49.6 28.172 11.78-2.99 17.282-12.201 19.795-19.377 6.766-19.524.77-46.142-15.096-66.28-20.498-26.126-77.86-69.52-148.635-47.658-30.223 9.337-54.957 27.735-71.521 53.068-13.722 21.025-21.7 47.137-21.874 71.488-.276 45.34 36.884 104.991 37.236 105.532a12.29 12.29 0 01-3.906 16.936l-.114.07a12.584 12.584 0 01-17.235-3.856c-1.677-2.728-41.356-66.33-41.036-118.732.445-56.8 34.961-124.589 111.154-148.125 35.218-10.88 72.229-9.051 107.058 5.229 26.995 11.136 51.979 29.57 68.497 50.62 21.097 26.867 28.53 62.037 18.926 89.487-6.403 18.3-19.66 30.926-37.276 35.44-45.913 11.68-63.9-17.726-77.018-39.177-13.47-21.99-20.92-32.301-44.202-25.11-12.783 3.95-20.659 10.277-23.971 19.369-4.507 12.428-.236 28.357 4.141 39.525l.137.35a100.946 100.946 0 0025.613 37.252c18.012 16.633 51.451 40.347 94.07 38.06a12.309 12.309 0 0113.048 11.522l.007.118a12.579 12.579 0 01-8.853 12.472z"
          fill="#db0000"
        />
        <path
          d="M194.068 352.83a12.616 12.616 0 01-9.87-1.031c-38.558-21.846-65.625-51.229-85.116-92.41l-.079-.255c-12.284-27.458-19.679-69.409-3.367-104.866 12.042-26.162 34.646-44.551 67.07-54.567 38.35-11.847 74.06-3.125 103.367 25.102a153.17 153.17 0 0131.984 45.299 12.516 12.516 0 01-23.03 9.803 128.26 128.26 0 00-26.967-37.76c-22.59-21.492-48.85-27.847-78.142-18.798-25.313 7.819-42.743 21.642-51.666 41.091-12.867 27.994-6.242 62.95 3.37 84.463 17.187 36.423 41.026 62.258 74.962 81.451a12.255 12.255 0 014.654 16.695l-.095.166a12.569 12.569 0 01-7.075 5.617z"
          fill="#db0000"
        />
        <circle
          cx="198.128"
          cy="213.768"
          fill="#e6e6e6"
          r="15.949"
        />
        <path
          d="M485.703 408.52a6.57 6.57 0 01-.984-.074 6.46 6.46 0 01-4.26-2.57l-8.761-11.878a6.499 6.499 0 011.373-9.09l74.899-55.24a6.5 6.5 0 019.09 1.374l8.76 11.878a6.5 6.5 0 01-1.372 9.09l-74.9 55.24a6.454 6.454 0 01-3.845 1.27z"
          fill="#db0000"
        />
        <path
          d="M538.897 364.044l-11.464-15.543a6 6 0 011.268-8.39l43.86-38.43a6 6 0 018.39 1.267l20.964 25.043a6 6 0 01-1.267 8.39l-53.361 28.93a6 6 0 01-8.39-1.267z"
          fill="#2f2e41"
        />
        <path
          d="M609.373 530.252h-14.758a6.508 6.508 0 01-6.5-6.5V400.737a6.508 6.508 0 016.5-6.5h14.758a6.508 6.508 0 016.5 6.5v123.015a6.508 6.508 0 01-6.5 6.5zM638.566 530.252h-14.76a6.508 6.508 0 01-6.5-6.5V400.737a6.508 6.508 0 016.5-6.5h14.76a6.508 6.508 0 016.5 6.5v123.015a6.508 6.508 0 01-6.5 6.5z"
          fill="#db0000"
        />
        <path
          d="M452.923 528.351h324.033v2.241H452.923z"
          fill="#3f3d56"
        />
        <circle
          cx="610.216"
          cy="195.742"
          fill="#db0000"
          r="53.519"
        />
        <path
          d="M693.072 467.175l-30.994-137.646v-32.521a8 8 0 00-8-8h-29.69v-20.913a8 8 0 00-8-8h-16a8 8 0 00-8 8v20.913h-29.69a8 8 0 00-8 8v171.087a7.968 7.968 0 00.584 2.993 7.834 7.834 0 007.377 6.206l125.424 4.182c7.496.197 10.897-9.553 4.99-14.301zM610.73 214.757c-3.305-.093-7.42-.208-10.588-2.524a8.131 8.131 0 01-3.2-6.07 5.471 5.471 0 011.86-4.495c1.657-1.399 4.076-1.726 6.678-.961l-2.7-19.727 1.983-.271 3.172 23.19-1.654-.759c-1.92-.88-4.552-1.328-6.188.055a3.513 3.513 0 00-1.152 2.896 6.143 6.143 0 002.38 4.527c2.467 1.802 5.747 2.035 9.466 2.139zM582.21 182.682h10.771v2H582.21zM616.21 182.682h10.771v2H616.21z"
          fill="#2f2e41"
        />
        <path
          d="M563.35 161.15c-9.507-5.762-13.985-16.691-13.993-27.5a39.637 39.637 0 0114.012-30.276c9.792-8.469 22.832-12.613 35.57-13.666 15.414-1.274 30.727 2.076 45.446 6.352 15.762 4.579 31.206 10.34 47.34 13.545 15.656 3.11 31.354 3.417 46.768-1.08 14.227-4.152 28.668-11.488 43.864-10.217 13.124 1.098 23.092 9.832 29.797 20.623 13.311 21.423 14.363 50.178 33.506 68.018 8.277 7.714 19.06 11.208 29.836 13.82 10.334 2.504 21.358 4.109 30.841 9.177 10.169 5.435 16.325 15.65 14.734 27.354-1.807 13.294-11.266 24.735-21.681 32.545a91.73 91.73 0 01-77.395 15.126c-27.632-7.331-50.087-28.746-79.527-30.089-16.424-.748-32.594 3.726-49.015 2.984-11.381-.514-23.63-4.721-31.557-13.185a24.709 24.709 0 01-6.757-14.537c-.18-1.906-3.181-1.924-3 0 1.125 11.957 9.927 20.789 20.43 25.644a56.355 56.355 0 0017.48 4.812c8.036.888 16.229.048 24.228-.829 8.455-.927 16.924-2.069 25.448-1.966a72.74 72.74 0 0122.837 4.061c14.106 4.838 26.813 12.865 40.288 19.116 13.043 6.05 26.7 9.802 41.146 9.895a96.211 96.211 0 0040.368-8.657c12.477-5.684 24.203-14.17 31.972-25.608 7.284-10.724 11.108-24.793 4.553-36.799-5.138-9.41-14.936-14.172-24.816-17.103-10.952-3.25-22.4-4.735-33.12-8.782a45.593 45.593 0 01-15.172-9.1 57.003 57.003 0 01-11.53-15.91c-6.138-11.974-9.275-25.124-14.444-37.491-4.788-11.456-11.34-23.003-21.893-30.063a37.244 37.244 0 00-19.058-6.151c-7.517-.34-14.988 1.386-22.133 3.56-7.561 2.302-14.94 5.176-22.558 7.294a81.672 81.672 0 01-24.668 2.946c-17.122-.557-33.662-5.644-49.872-10.772-28.395-8.983-59.976-18.6-88.524-4.753-11.124 5.396-20.546 14.448-24.519 26.335-3.515 10.519-3.029 22.787 2.39 32.587a30.455 30.455 0 0010.863 11.33c1.655 1.003 3.165-1.59 1.514-2.59z"
          fill="#2f2e41"
        />
        <path
          d="M646.088 428.594h-14.76a6.508 6.508 0 01-6.5-6.5V336.84a6.508 6.508 0 016.5-6.5h14.76a6.508 6.508 0 016.5 6.5v85.254a6.508 6.508 0 01-6.5 6.5z"
          fill="#db0000"
        />
        <path
          d="M589.116 147.907a12.736 12.736 0 0021.659 7.201l-2.507-.662a21.205 21.205 0 0037.151 4.397c1.13-1.57-1.471-3.069-2.59-1.514a18.21 18.21 0 01-31.668-3.68 1.522 1.522 0 00-2.507-.662 9.758 9.758 0 01-16.645-5.877c-.268-1.904-3.159-1.094-2.893.797z"
          fill="#2f2e41"
        />
        <path
          d="M494.552 360.893l-.313.212-5.807-8.567a5.99 5.99 0 00-8.32-1.597l-18.15 12.305a5.99 5.99 0 00-1.596 8.319l31.861 46.998a5.99 5.99 0 008.32 1.597l18.15-12.305a5.99 5.99 0 001.596-8.319l-21.92-32.333.313-.212z"
          data-name="Path 33"
          fill="#3f3d56"
        />
        <path
          d="M481.485 353.166l-2.293 1.554a2.057 2.057 0 01.014 3.415l-10.064 6.823a2.057 2.057 0 01-3.167-1.277l-2.142 1.452a4.33 4.33 0 00-1.154 6.014l30.793 45.422a4.33 4.33 0 006.014 1.154l17.651-11.966a4.33 4.33 0 001.155-6.014l-30.794-45.423a4.33 4.33 0 00-6.013-1.154z"
          data-name="Path 34"
          fill="#fff"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      data-name="Layer 1"
      height="265"
      viewBox="0 0 924.256 530.591"
      width="462"
    >
      <path
        d="M16.126 159.262a12.416 12.416 0 01-15.82-14.588C7.53 114.622 34.998 41.128 135.064 10.217c43.345-13.39 84.914-13.619 123.5-.685 31.744 10.596 52.817 27.178 62.897 35.193a12.313 12.313 0 012.017 17.296l-.076.095a12.603 12.603 0 01-17.56 1.994c-18.264-14.39-73.685-58.105-163.508-30.358-87.706 27.093-111.53 90.466-117.71 116.393a12.231 12.231 0 01-8.498 9.117z"
        fill="#db0000"
      />
      <path
        d="M288.866 329.481a12.346 12.346 0 01-3.067.577c-78.175 4.21-127.914-60.736-129.956-63.443l-.378-.625c-1.247-1.934-31.342-47.723-18.093-84.088 6.086-16.625 19.544-28.294 40.116-34.65 19.133-5.91 34.79-4.162 48.116 5.354 10.926 7.753 17.93 19.315 24.717 30.48 14.116 23.182 22.767 34.977 49.6 28.172 11.78-2.99 17.282-12.201 19.795-19.377 6.766-19.524.77-46.142-15.096-66.28-20.498-26.126-77.86-69.52-148.635-47.658-30.223 9.337-54.957 27.735-71.521 53.068-13.722 21.025-21.7 47.137-21.874 71.488-.276 45.34 36.884 104.991 37.236 105.532a12.29 12.29 0 01-3.906 16.936l-.114.07a12.584 12.584 0 01-17.235-3.856c-1.677-2.728-41.356-66.33-41.036-118.732.445-56.8 34.961-124.589 111.154-148.125 35.218-10.88 72.229-9.051 107.058 5.229 26.995 11.136 51.979 29.57 68.497 50.62 21.097 26.867 28.53 62.037 18.926 89.487-6.403 18.3-19.66 30.926-37.276 35.44-45.913 11.68-63.9-17.726-77.018-39.177-13.47-21.99-20.92-32.301-44.202-25.11-12.783 3.95-20.659 10.277-23.971 19.369-4.507 12.428-.236 28.357 4.141 39.525l.137.35a100.946 100.946 0 0025.613 37.252c18.012 16.633 51.451 40.347 94.07 38.06a12.309 12.309 0 0113.048 11.522l.007.118a12.579 12.579 0 01-8.853 12.472z"
        fill="#db0000"
      />
      <path
        d="M194.068 352.83a12.616 12.616 0 01-9.87-1.031c-38.558-21.846-65.625-51.229-85.116-92.41l-.079-.255c-12.284-27.458-19.679-69.409-3.367-104.866 12.042-26.162 34.646-44.551 67.07-54.567 38.35-11.847 74.06-3.125 103.367 25.102a153.17 153.17 0 0131.984 45.299 12.516 12.516 0 01-23.03 9.803 128.26 128.26 0 00-26.967-37.76c-22.59-21.492-48.85-27.847-78.142-18.798-25.313 7.819-42.743 21.642-51.666 41.091-12.867 27.994-6.242 62.95 3.37 84.463 17.187 36.423 41.026 62.258 74.962 81.451a12.255 12.255 0 014.654 16.695l-.095.166a12.569 12.569 0 01-7.075 5.617z"
        fill="#db0000"
      />
      <circle
        cx="198.128"
        cy="213.768"
        fill="#e6e6e6"
        r="15.949"
      />
      <path
        d="M485.703 408.52a6.57 6.57 0 01-.984-.074 6.46 6.46 0 01-4.26-2.57l-8.761-11.878a6.499 6.499 0 011.373-9.09l74.899-55.24a6.5 6.5 0 019.09 1.374l8.76 11.878a6.5 6.5 0 01-1.372 9.09l-74.9 55.24a6.454 6.454 0 01-3.845 1.27z"
        fill="#db0000"
      />
      <path
        d="M538.897 364.044l-11.464-15.543a6 6 0 011.268-8.39l43.86-38.43a6 6 0 018.39 1.267l20.964 25.043a6 6 0 01-1.267 8.39l-53.361 28.93a6 6 0 01-8.39-1.267z"
        fill="#2f2e41"
      />
      <path
        d="M609.373 530.252h-14.758a6.508 6.508 0 01-6.5-6.5V400.737a6.508 6.508 0 016.5-6.5h14.758a6.508 6.508 0 016.5 6.5v123.015a6.508 6.508 0 01-6.5 6.5zM638.566 530.252h-14.76a6.508 6.508 0 01-6.5-6.5V400.737a6.508 6.508 0 016.5-6.5h14.76a6.508 6.508 0 016.5 6.5v123.015a6.508 6.508 0 01-6.5 6.5z"
        fill="#db0000"
      />
      <path
        d="M452.923 528.351h324.033v2.241H452.923z"
        fill="#3f3d56"
      />
      <circle
        cx="610.216"
        cy="195.742"
        fill="#db0000"
        r="53.519"
      />
      <path
        d="M693.072 467.175l-30.994-137.646v-32.521a8 8 0 00-8-8h-29.69v-20.913a8 8 0 00-8-8h-16a8 8 0 00-8 8v20.913h-29.69a8 8 0 00-8 8v171.087a7.968 7.968 0 00.584 2.993 7.834 7.834 0 007.377 6.206l125.424 4.182c7.496.197 10.897-9.553 4.99-14.301zM610.73 214.757c-3.305-.093-7.42-.208-10.588-2.524a8.131 8.131 0 01-3.2-6.07 5.471 5.471 0 011.86-4.495c1.657-1.399 4.076-1.726 6.678-.961l-2.7-19.727 1.983-.271 3.172 23.19-1.654-.759c-1.92-.88-4.552-1.328-6.188.055a3.513 3.513 0 00-1.152 2.896 6.143 6.143 0 002.38 4.527c2.467 1.802 5.747 2.035 9.466 2.139zM582.21 182.682h10.771v2H582.21zM616.21 182.682h10.771v2H616.21z"
        fill="#2f2e41"
      />
      <path
        d="M563.35 161.15c-9.507-5.762-13.985-16.691-13.993-27.5a39.637 39.637 0 0114.012-30.276c9.792-8.469 22.832-12.613 35.57-13.666 15.414-1.274 30.727 2.076 45.446 6.352 15.762 4.579 31.206 10.34 47.34 13.545 15.656 3.11 31.354 3.417 46.768-1.08 14.227-4.152 28.668-11.488 43.864-10.217 13.124 1.098 23.092 9.832 29.797 20.623 13.311 21.423 14.363 50.178 33.506 68.018 8.277 7.714 19.06 11.208 29.836 13.82 10.334 2.504 21.358 4.109 30.841 9.177 10.169 5.435 16.325 15.65 14.734 27.354-1.807 13.294-11.266 24.735-21.681 32.545a91.73 91.73 0 01-77.395 15.126c-27.632-7.331-50.087-28.746-79.527-30.089-16.424-.748-32.594 3.726-49.015 2.984-11.381-.514-23.63-4.721-31.557-13.185a24.709 24.709 0 01-6.757-14.537c-.18-1.906-3.181-1.924-3 0 1.125 11.957 9.927 20.789 20.43 25.644a56.355 56.355 0 0017.48 4.812c8.036.888 16.229.048 24.228-.829 8.455-.927 16.924-2.069 25.448-1.966a72.74 72.74 0 0122.837 4.061c14.106 4.838 26.813 12.865 40.288 19.116 13.043 6.05 26.7 9.802 41.146 9.895a96.211 96.211 0 0040.368-8.657c12.477-5.684 24.203-14.17 31.972-25.608 7.284-10.724 11.108-24.793 4.553-36.799-5.138-9.41-14.936-14.172-24.816-17.103-10.952-3.25-22.4-4.735-33.12-8.782a45.593 45.593 0 01-15.172-9.1 57.003 57.003 0 01-11.53-15.91c-6.138-11.974-9.275-25.124-14.444-37.491-4.788-11.456-11.34-23.003-21.893-30.063a37.244 37.244 0 00-19.058-6.151c-7.517-.34-14.988 1.386-22.133 3.56-7.561 2.302-14.94 5.176-22.558 7.294a81.672 81.672 0 01-24.668 2.946c-17.122-.557-33.662-5.644-49.872-10.772-28.395-8.983-59.976-18.6-88.524-4.753-11.124 5.396-20.546 14.448-24.519 26.335-3.515 10.519-3.029 22.787 2.39 32.587a30.455 30.455 0 0010.863 11.33c1.655 1.003 3.165-1.59 1.514-2.59z"
        fill="#2f2e41"
      />
      <path
        d="M646.088 428.594h-14.76a6.508 6.508 0 01-6.5-6.5V336.84a6.508 6.508 0 016.5-6.5h14.76a6.508 6.508 0 016.5 6.5v85.254a6.508 6.508 0 01-6.5 6.5z"
        fill="#db0000"
      />
      <path
        d="M589.116 147.907a12.736 12.736 0 0021.659 7.201l-2.507-.662a21.205 21.205 0 0037.151 4.397c1.13-1.57-1.471-3.069-2.59-1.514a18.21 18.21 0 01-31.668-3.68 1.522 1.522 0 00-2.507-.662 9.758 9.758 0 01-16.645-5.877c-.268-1.904-3.159-1.094-2.893.797z"
        fill="#2f2e41"
      />
      <path
        d="M494.552 360.893l-.313.212-5.807-8.567a5.99 5.99 0 00-8.32-1.597l-18.15 12.305a5.99 5.99 0 00-1.596 8.319l31.861 46.998a5.99 5.99 0 008.32 1.597l18.15-12.305a5.99 5.99 0 001.596-8.319l-21.92-32.333.313-.212z"
        data-name="Path 33"
        fill="#3f3d56"
      />
      <path
        d="M481.485 353.166l-2.293 1.554a2.057 2.057 0 01.014 3.415l-10.064 6.823a2.057 2.057 0 01-3.167-1.277l-2.142 1.452a4.33 4.33 0 00-1.154 6.014l30.793 45.422a4.33 4.33 0 006.014 1.154l17.651-11.966a4.33 4.33 0 001.155-6.014l-30.794-45.423a4.33 4.33 0 00-6.013-1.154z"
        data-name="Path 34"
        fill="#fff"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
